examples/unstructured_profilers.ipynb

```diff
@@ -179,7 +179,7 @@
     "profile_options = dp.ProfilerOptions()\n",
     "\n",
     "# Setting multiple options via set\n",
-    "profile_options.set({ \"vocab.is_enabled\": False, \"is_case_sensitive\": True })\n",
+    "profile_options.set({ \"*.vocab.is_enabled\": False, \"*.is_case_sensitive\": True })\n",
```

Comment:
Bug in notebook from change made in previous PR where options no longer search unless specified with `*.`